_posts/2017-11-07-android-rpc-introduction.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title:  "Test Computation Workloads on Android Phones"
```

Comment:
on Mobile Phones(we can use Android as an example, and add link to iOS RPC as well)